shared/js/background/classes/site.es6.js

```diff
@@ -43,8 +44,8 @@ class Site {
             this.grade.setHttps(true, true)
         }
 
-        // set isSpecialDomain when the site is created
-        this.isSpecialDomain = this.specialDomain()
+        // set specialDomainName when the site is created
+        this.specialDomainName = this.getSpecialDomain()
```

Comment:
Sorry, couldn't resist renaming these. Variables that pretend to be booleans and method names that pretend to not be method names are a pet peeve of mine.